doc/glossary.rst

```diff
@@ -1479,8 +1479,9 @@ functions or non-estimator constructors.
 
         ``n_jobs`` is an int, specifying the maximum number of concurrently
         running jobs.  If set to -1, all CPUs are used. If 1 is given, no
-        parallel computing code is used at all.  For n_jobs below -1, (n_cpus +
```

Comment:
The statement already here is not quite true. Better to say no job-level parallelism or something, and perhaps to note that other parallelism may be done by the numerical processing libraries (as per the FAQ's coverage of this)...